styles/styles.css

```diff
@@ -524,6 +524,55 @@ main .section.brands-top>.section-container {
   padding: 0 1rem;
 }
 
+/* section - social */
+main .section.social-media {
+  padding: 4rem 0;
+  background-color: var(--transparent-blue-light-color);
+}
+
+main .section.social-media .section-container {
+  padding: 0 1rem;
+}
+
+main .section.social-media .default-content-wrapper svg {
+  display: none;
+}
+
+main .section.social-media a.button.primary {
+  background: unset;
+  justify-content: unset;
+  align-items: center;
+  display: flex;
+  line-height: 2.5rem;
+  color: var(--light-black);
+  font-weight: var(--font-weight-medium);
+  font-size: var(--body-font-size-m);
+  padding-left: 0;
+  padding-right: 0;
+  letter-spacing: -0.006rem;
```

Comment:
Nit:  this really needed? It's less than 0.1px

Reply:
This keeps the same with Sunstar-engineering site, we can remove it.